### ISSUE TITLE
composer model trans to pythia problem

### ISSUE THREAD

**Issue by rzr002** [princeton-nlp/LLM-Shearing#64]
I've encountered the same issue; I tried to modify the Pythia model to a Composer model by imitating your llama rewrite, and although it runs successfully, the output logits are quite different. Could you please expedite the updates for Pythia or Mistral?
